Projects/1-Beginner/Bin2Dec-App.md

```
@@ -45,3 +45,5 @@ Try not to view this until you've developed your own solution:
 -   [Binary to Decimal converter using React](https://github.com/geoffctn/Bin2Dec)
 -   [Matrix-like Binary to Decimal converter using Angular](https://github.com/ZangiefWins/MatrixBin2Dec)
     -   [Live preview on heroku](https://matrix-bin2dec.herokuapp.com/)
+- [Binary to Decimal converter using Blazor WebAssembly](https://github.com/mlgwan/Bin2Dec)
+    -   [Live preview](https://projects.ccl97.com/bin2dec)  
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)